envs/tests/test_envs.py

```diff
@@ -55,36 +55,37 @@ def test_env_performance(make_env, env_type, verbose=False):
     log.debug('Took %.3f sec to collect %d frames on one CPU, %.1f FPS', t.experience, total_num_frames, fps)
     log.debug('Avg. reset time %.3f s', t.reset / num_resets)
     log.debug('Timing: %s', t)
-
     env.close()
 
 
-def test_multi_env_performance(make_env, env_type, num_envs, num_workers, total_num_frames=100000):
-    t = Timing()
-    frames = 0
-
-    with t.timeit('init'):
-        multi_env = MultiEnv(num_envs, num_workers, make_env, stats_episodes=100)
-
-    with t.timeit('first_reset'):
-        multi_env.reset()
-
-    next_print = print_step = 10000
-
-    with t.timeit('experience'):
-        while frames < total_num_frames:
-            _, rew, done, info = multi_env.step([multi_env.action_space.sample()] * num_envs)
-            frames += num_env_steps(info)
-            if frames > next_print:
-                log.info('Collected %d frames of experience...', frames)
-                next_print += print_step
-
-    fps = total_num_frames / t.experience
-    log.debug('%s performance:', env_type)
-    log.debug('Took %.3f sec to collect %d frames in parallel, %.1f FPS', t.experience, total_num_frames, fps)
-    log.debug('Timing: %s', t)
-
-    multi_env.close()
+# def test_multi_env_performance(make_env, env_type, num_envs, num_workers, total_num_frames=1000):
```

Comment:
This test relied on the old class MultiEnv
Now we can rewrite it using DummySampler class
Or just remove this test completely

Reply:
Sure, probably thought I would rewrite it